packages/docs/content/components/drawer.mdx

```diff
@@ -14,9 +14,9 @@ medicare:
   figmaNodeId: 6-91
 ---
 
-On large screens it's fixed to the side of the screen, and on smaller screens it overlays the entire screen.
+A `Drawer` is a component that provides a space for displaying medium to long-form content. It is designed to present information that may be too lengthy or too specific to be included on the page by default. The content can include a mix of text, HTML elements, or other components, allowing for flexibility in displaying information such as forms, images, links, or lists.
 
-Render the drawer below the toggle button that triggers it. This way the markup remains semantically sound and screen reader friendly.
+The `HelpDrawer` extends the `Drawer` by adding analytics capabilities, which can help track user interactions, making it a great choice for situations where gathering insights on how users engage with help content is important.
```

Comment:
I like this explanation and how it justifies the name: "gathering insights on how users engage with help content"